frontend/src/views/DriveVerifyView.vue

```diff
@@ -0,0 +1,172 @@
+<template>
+  <div>
+    <main-form
+      v-if="!submitted"
+      :title="$t('common.confirm_drive')"
```

Comment:
```suggestion
      :title="{{ $t('common.confirm_drive') }}"
```
or am I missing something?

Reply:
`:title="$t(...)"` is equivalent to `title="{{ $t(...) }}"`, there is a colon before `title`